<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameconfigEditorv1</class>
 <widget class="QWidget" name="GameconfigEditorv1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="sectionLabel">
     <property name="text">
      <string>Details</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QListWidget" name="sectionList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Players</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Categories</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QStackedWidget" name="detailsWidget">
     <widget class="QWidget" name="playersPage">
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="6" column="0">
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="5">
          <widget class="QToolButton" name="rmPlr">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="3">
          <widget class="QToolButton" name="downPlr">
           <property name="text">
            <string>D</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QToolButton" name="upPlr">
           <property name="text">
            <string>U</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QToolButton" name="addPlr">
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="5" column="2">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>311</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Display Name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="plrAnim"/>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>P1 Animation</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="plrName"/>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="plrNameDisplay"/>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Player Name</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QCheckBox" name="hasP2">
         <property name="text">
          <string>Has Player 2</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="6">
        <widget class="QListWidget" name="plrList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>194</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>P2 Animation</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="plr2Anim">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="categoriesPage">
      <layout class="QGridLayout" name="gridLayout_13">
       <item row="0" column="2">
        <widget class="QLineEdit" name="scnName"/>
       </item>
       <item row="4" column="2">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>226</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Folder</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="scnFolder"/>
       </item>
       <item row="6" column="0">
        <layout class="QGridLayout" name="gridLayout_12">
         <item row="0" column="0">
          <widget class="QToolButton" name="addScn">
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QToolButton" name="rmScn">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QToolButton" name="upScn">
           <property name="text">
            <string>U</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="3">
          <widget class="QToolButton" name="downScn">
           <property name="text">
            <string>D</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="scnID"/>
       </item>
       <item row="0" column="0" rowspan="5">
        <widget class="QTreeView" name="scnTree">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QCheckBox" name="scnHighlighted">
         <property name="toolTip">
          <string>if checked this will be the highlighted text on dev menu (used for act 1's that have entire names e.g. &quot;Green Hill Zone 1&quot; would be highlighted, and &quot;2&quot; (GHZ2) wouldn't be)</string>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Highlighted on Menu</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
